feature-libs/asm/components/customer-emulation/customer-emulation.component.ts

```diff
@@ -1,20 +1,40 @@
-import { Component, OnDestroy, OnInit } from '@angular/core';
+import {
```

Comment:
I would probably improve the design to encapsulate the cart binding functionality in its own dedicated component.  Let's discuss.